.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "2.7"
+  - "3.6"
+  - "3.6-dev"
+  - "3.7-dev"
+  - "nightly"
+script: make GREP=/bin/grep LN=/bin/ln MKDIR=/bin/mkdir PYTHON_VERSION=3.6.0 SH=/bin/sh TAR=/bin/tar test
```

Comment:
No, that wasn't supposed to be in the pull request at all. Should I create a new PR, or do you want to remove this manually?